lib/index.js

```diff
@@ -0,0 +1,207 @@
+const sdk = require('postman-collection'),
+  errors = require('./assets/error'),
+  fs = require('fs'),
+  path = require('path'),
+  _ = require('lodash'),
+  fetch = require('node-fetch'),
+  languageMap = require('./assets/languageMap.json'),
+  generator = require('../sdkgen/nodejs-request').generate; // TODO fix language mapping. this is temporary
```

Comment:
Please add these todos to the project as tasks. These should be fixed before we plan to add more sdkgens.

Reply:
https://github.com/postmanlabs/postman-code-generators/blob/develop/lib/index.js
You can follow similar pattern followed here^.